L10nSnapshotTests/L10nBaseSnapshotTests.swift

```diff
@@ -7,7 +7,7 @@ import XCTest
 
 let testPageBase = "http://www.example.com"
 let loremIpsumURL = "\(testPageBase)"
-
+let TIMEOUT: TimeInterval = 15
```

Comment:
I created these `TIMEOUT` constants so that we are using the same timeout throughout.
https://github.com/mozilla-mobile/firefox-ios/blob/main/Tests/XCUITests/BaseTestCase.swift#L18

Reply:
i added that there because from looking at the tests, they inherit from this file, not `BaseTestCase`, which is why there's some duplicated code here.

If there's a higher scope or if the `TIMEOUT` is being pulled in, for sure I'll remove it.

Is `TIMEOUT` set somewhere else in the solution where it's getting imported here?

Reply:
yeah, the L10n tests inherit from that L10nBaseTestCase not the other one... Honestly, not sure why it was that way, I guess we could change that at some point but not in this PR